src/components/vm/nics/nicBody.tsx

```diff
@@ -139,18 +171,20 @@ export const NetworkTypeAndSourceRow = ({
         ];
     } else {
         availableNetworkTypes = [
-            ...virtualNetwork,
+            {
+                name: 'passt',
+                desc: 'Userspace PASST stack',
+                detailParagraph: _("Recommended method to provide a virtual LAN with NAT to the outside world. Supports port forwarding.")
+            },
             {
                 name: 'user',
-                desc: 'Userspace SLIRP stack',
-                detailParagraph: _("Provides a virtual LAN with NAT to the outside world.")
+                desc: 'Userspace SLIRP stack (legacy)',
+                detailParagraph: _("Legacy method to provide a virtual LAN with NAT to the outside world.")
```

Comment:
Why do we display "Userspace SLIRP stack" and such to begin with? Feels a bit weird to me to have it displayed in the select vs just writing "slirp" and "passt". We can in the select dropdown have a description for each entry to make it easier, which would look like this: https://www.patternfly.org/components/menus/menu#with-descriptions